Week2/homework/maartjes-work.js

```diff
@@ -46,7 +46,15 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
+ return maartjesTasks.map(task => task.duration)
+ //change from seconds to hours
+ .map(durationMinutes => durationMinutes/60)
+ //filter less than 2 hours
+ .filter(durationHour => durationHour >= 2)
+ //calculate the billing and sum up
+ .map(durationHour => durationHour * 20)
```

Comment:
You put hourlyRate as a variable but then you hard-coded it any way 😄 
Nicely done otherwise!